docs/content/users/usage/commands.md

```diff
@@ -304,7 +304,7 @@ Flags:
 * `--bind-all-interfaces`: Bind host ports on all interfaces, not only on localhost network interface.
 * `--composer-root`: Overrides the default Composer root directory for the web service.
 * `--composer-root-default`: Unsets a web service Composer root directory override.
-* `--composer-version`: Specify override for Composer version in the web container. This may be `""`, `"1"`, `"2"`, `"2.2"`, `"stable"`, `"preview"`, `"snapshot"`, or a specific version.
+* `--composer-version`: Specify override for Composer version in the web container. This may be `""`, `"1"`, `"2"`, `"2.2"`, `"stable"`, `"preview"`, `"snapshot"`, or a specific version. (Note that if your project composer build requires `composer/composer` then the version you require there will be used instead of the version specified here.)
```

Comment:
I guess this should also be mentioned in the YAML template?

https://github.com/ddev/ddev/blob/1870c38c18ef91f615902d967606bd802b6812d1/pkg/ddevapp/templates.go#L60-L69

Reply:
I'm reluctant to add more information there. Let's leave it the way it is for now.

Reply:
Okay.